tests/wpt/README.md

```diff
@@ -1,246 +1,6 @@
-This folder contains the web platform tests and the
-code required to integrate them with Servo. 
-To learn how to write tests, go [here](http://web-platform-tests.org/writing-tests/index.html).
+# Web tests
```

Comment:
This is the proper name of a project so all caps, I think:

```suggestion
# Web Platform Tests
```

Reply:
not all of the tests in tests/wpt are Web Platform Tests, only those in tests/wpt/tests. i’ve updated it to mention both in bf938ae761742a4a7ecdf697cbbda17ebc059e9c